fms/utils/generation.py

```diff
@@ -6,25 +6,36 @@
 import torch.nn.functional as F
 
 
-def __prepare_list_input(
-    input_ids_list: List[torch.Tensor], model: Union[Callable, torch.nn.Module]
+def make_padded(
```

Comment:
maybe something like `pad_input_ids(input_ids: List[Tensor], min_length=0)` might be clearer/simpler?

Reply:
updated to `pad_input_ids(input_ids_list: List[torch.Tensor], pad_to_min_length: int = 0)`